lib/plugins/aws/deploy/lib/create-stack.js

```diff
@@ -88,6 +88,21 @@ module.exports = {
                 this.provider.disableTransferAccelerationForCurrentDeploy();
               }
             }
+
+            const stacksExist = Array.isArray(data.Stacks);
+            const singleStack = stacksExist && data.Stacks.length <= 1;
+            const shouldProcessStacks =
+              singleStack && data.Stacks[0].StackStatus === 'REVIEW_IN_PROGRESS';
+            if (shouldProcessStacks) {
```

Comment:
It's nice to be able to clearly recognize the situation where an error occurs using `inactiveStateNames`!

Since the business logic to verify the error is as follows, the final modified code will be as follows.

1. The number of `data.Stacks` arrays is less than 1.
2. Have an abnormal Stack Status (ex: `REVIEW_IN_PROGRESS`);
```suggestion
            const inactiveStateNames = new Set(['REVIEW_IN_PROGRESS']);
            const stacksExist = Array.isArray(data.Stacks);
            const isSingleStack = stacksExist && data.Stacks.length <= 1;
            const shouldCheckStackStatus = isSingleStack && inactiveStateNames.has(data.Stacks[0].StackStatus);
            if (shouldCheckStackStatus) {
```
Here, when the number of `data.Stacks` array exceeds 1, we thought that CloudFormation Stack might be running normally, so we added that condition.